src/pdlpack/pdlpack.py

```diff
@@ -1295,6 +1295,7 @@ def install_check(schema, platform, sugar_schema, madlib_schema, args):
 	    session.exec_query("DROP USER IF EXISTS %s;" % (test_user), True)
 	session.exec_query("CREATE USER %s;" % (test_user), True)
 	session.exec_query("GRANT ALL ON SCHEMA %s TO %s;" %(schema, test_user), True)
+	session.exec_query("GRANT ALL ON SCHEMA %s TO %s;" %(madlib_schema, test_user), True)
```

Comment:
We should not allow the user to mess around with the MADlib schema. Does your module actually depend on modifying the MADlib schema? If so, we need to discuss this.